apps/app/src/views/routes/Settings/components/users/UserItemActions.tsx

```diff
@@ -63,6 +83,54 @@ export const UserItemActions: React.FC<UserItemActionsProps> = ({ user }) => {
           <MenuItem icon={<FiEdit fontSize="1.2em" />} onClick={onOpen}>
             Edit Roles
           </MenuItem>
+          <MenuItem
+            icon={<FiTrash fontSize="1.3em" color="red" />}
+            disabled={deleteLoading}
+            onClick={async () => {
+              const decision = await confirmWrapper('Remove user?', {
+                description: <Text mb={2}>This will remove user from organization.</Text>,
```

Comment:
Is there a reason we're overloading the styling here?

Reply:
this is copy paste from delete invite that was approved

Reply:
Is there an issue with it?